src/alto/app/estimator.py

```diff
@@ -0,0 +1,76 @@
+"""

```

Comment:
Do you mind explaining what do the folder names 'app' and 'model' mean under the src/alto path? What modules and functions do we plan to include under these two folders? thks.

Reply:
I personally do not know what the plan is for the 'model' folder; I was not the one who created it. However, the 'app' folder would be a good place to put client-side code that can be run as standalone utilities. `estimator.py` is an example of one such utility, which can be run from a CLI to get throughput information (in addition to defining methods that can be used in code from elsewhere). Please let me know if you have any suggestions regarding this design.

Reply:
> Do you mind explaining what do the folder names 'app' and 'model' mean under the src/alto path? What modules and functions do we plan to include under these two folders? thks.

`src/alto/app` is supposed to be used for some simple applications that can use the ALTO client library to interact with the ALTO server and do some more complex business logic after receiving raw ALTO responses. There is a simple principle: if a function will be used by multiple applications, it should be considered as a part of the ALTO client library.

`src/alto/model` is used to define data models for ALTO protocol message serialization/deserialization. As this part can be commonly used by both client and server, @emiapwil and I considered moving it to `src/alto/common` later.

Personally, I think the `do_request` function should be moved into the `alto.client.Client` class.

Reply:
That's a great explanation and makes sense, thank you @fno2010 .

I agree that `do_request` should be moved into `alto.client.Client` class

Reply:
Could we copy this explanation into the [README.rst](https://github.com/openalto/alto/tree/task2-arch-fix-untested#readme) file, explaining the structure of the source code and what each folder is used for?

Reply:
Sounds good. But maybe we can start another PR to do this. No need to make a single PR too large.

Reply:
I agree, I just created a new separate ticket referencing this comment, thanks.